### ISSUE TITLE
Script not working with "Enhancer for YouTube" after v5.3

### ISSUE THREAD

**Issue by hugolinhares** [EvHaus/youtube-hide-watched#166]
I know that the extension "Enhancer for YouTube" is not oficially suported, but the script used to work with it before the 5.3 update. If possible could you check?

**Comment by EvHaus** [comment]
Could you be more specific about what doesn't work? I just tried it and at least in the Subscriptions page, watched videos still get hidden.

**Comment by hugolinhares** [comment]
Well that is strange. It doesn"t work in any page, including the Subscriptions page. The buttons on the top bar are missing too. Did you tried with extension Enhancer for YouTube™ (https://chrome.google.com/webstore/detail/enhancer-for-youtube/ponfpcnoihfmfllpaingbgckeeldkhle)? Thanks again.

**Comment by EvHaus** [comment]
Yeah, that's the one I used. Why do you think the extension is causing the script to break? Does uninstalling "Enhancer for YouTube" make it work for you?

**Comment by hugolinhares** [comment]
I'm actually runing the script in the "Enhancer for YouTube" extension. I'm not using TamperMonkey. If you go to EY options, there's one field where you can paste custons scripts. Before the 5.3 update I could run the script only with "Enhancer for YouTube", after the update it's not working anymore. As sad early, I know that this extension is not oficially suported, but the script used to work with it. If you could make compatible again, It would be awesome. Thanks for the attention and effort.

**Comment by jas32096** [comment]
Chances are your extension doesn't support `GM_registerMenuCommand`. You could try switching to using something that supports it or using my fork which removes it, as a dependency https://github.com/EvHaus/youtube-hide-watched/pull/163

**Comment by hugolinhares** [comment]
Thanks for the answer. I have tested your fork, but unfortunately it didn't work either.
